features/filesystem/fat/FATFileSystem.cpp

```diff
@@ -35,39 +35,27 @@
 static int fat_error_remap(FRESULT res)
 {
     switch(res) {
-        case FR_OK:                     /* (0) Succeeded */
-            return 0;                   /* no error */
-        case FR_DISK_ERR:               /* (1) A hard error occurred in the low level disk I/O layer */
-        case FR_NOT_READY:              /* (3) The physical drive cannot work */
-            return -EIO;                /* I/O error */
-        case FR_NO_FILE:                /* (4) Could not find the file */
-        case FR_NO_PATH:                /* (5) Could not find the path */
-        case FR_INVALID_NAME:           /* (6) The path name format is invalid */
-        case FR_INVALID_DRIVE:          /* (11) The logical drive number is invalid */
-        case FR_NO_FILESYSTEM:          /* (13) There is no valid FAT volume */
-            return -ENOENT;             /* No such file or directory */
-        case FR_DENIED:                 /* (7) Access denied due to prohibited access or directory full */
-            return -EACCES;             /* Permission denied */
-        case FR_EXIST:                  /* (8) Access denied due to prohibited access */
-            return -EEXIST;             /* File exists */
-        case FR_WRITE_PROTECTED:        /* (10) The physical drive is write protected */
-        case FR_LOCKED:                 /* (16) The operation is rejected according to the file sharing policy */
-            return -EACCES;             /* Permission denied */
-        case FR_INVALID_OBJECT:         /* (9) The file/directory object is invalid */
-            return -EFAULT;             /* Bad address */
-        case FR_NOT_ENABLED:            /* (12) The volume has no work area */
-            return -ENXIO;              /* No such device or address */
-        case FR_NOT_ENOUGH_CORE:        /* (17) LFN working buffer could not be allocated */
-            return -ENOMEM;             /* Not enough space */
-        case FR_TOO_MANY_OPEN_FILES:    /* (18) Number of open files > _FS_LOCK */
-            return -ENFILE;             /* Too many open files in system */
-        case FR_INVALID_PARAMETER:      /* (19) Given parameter is invalid */
-            return -ENOEXEC;            /* Exec format error */
-        case FR_INT_ERR:                /* (2) Assertion failed */
-        case FR_MKFS_ABORTED:           /* (14) The f_mkfs() aborted due to any parameter error */
-        case FR_TIMEOUT:                /* (15) Could not get a grant to access the volume within defined period */
-        default:                        /* Bad file number */
-            return -EBADF;
+        case FR_OK:                   return 0;           // (0) Succeeded
```

Comment:
can you fix the formatting (as it was)

```
case FR_OK:                   
    return 0;
```

Reply:
I kind of like the new style, makes it more compressed and a bit easier to read. Not sure of the precedence here though just my 2 cents.

Reply:
@0xc0170, https://github.com/ARMmbed/mbed-os/pull/6014 will update this anyways won't it?

Reply:
@geky Just because a PR fixes style issues, doesn't mean we now get to freely make them ;)

@kegilbert I believe the new line is already a precedent. I also think that if someone were doing a quick scan of the code, the indented new line sticks out, whereas this formatting is easy to miss (imo).

Reply:
Feel free to update

Reply:
Fair enough 
![image](https://user-images.githubusercontent.com/1060940/36607483-60228186-188c-11e8-9d08-054e83bd92af.png)